be/src/olap/utils.cpp

```diff
@@ -1104,11 +1104,11 @@ OLAPStatus Mutex::lock() {
 }
 
 OLAPStatus Mutex::trylock() {
-    if (0 != pthread_mutex_trylock(&_lock)) {
-        VLOG(3) << "failed to got the mutex lock. err=" << strerror(errno);
+    int rv = pthread_mutex_trylock(&_lock);
+    if (rv != 0) {
+        VLOG(3) << "failed to got the mutex lock. error=" << strerror(rv);
```

Comment:
use strerror_r to make it thread safe

Reply:
The return error is defined. So it is thread safe.